mmv1/third_party/terraform/resources/resource_bigtable_instance.go

```diff
@@ -85,6 +85,31 @@ func resourceBigtableInstance() *schema.Resource {
 							Computed:    true,
 							Description: `Describes the Cloud KMS encryption key that will be used to protect the destination Bigtable cluster. The requirements for this key are: 1) The Cloud Bigtable service account associated with the project that contains this cluster must be granted the cloudkms.cryptoKeyEncrypterDecrypter role on the CMEK key. 2) Only regional keys can be used and the region of the CMEK key must match the region of the cluster. 3) All clusters within an instance must use the same CMEK key. Values are of the form projects/{project}/locations/{location}/keyRings/{keyring}/cryptoKeys/{key}`,
 						},
+						"autoscaling_config": {
+							Type:        schema.TypeList,
+							Optional:    true,
+							MaxItems:    1,
+							Description: "A list of Autoscaling configurations. Only one element is used and allowed.",
```

Comment:
Is there a reason why a list is used here but exactly one element is required?

Reply:
Yea it's the only way to nest a object into the schema in terraform. More details is in yaqs. See internal design doc.